### PR TITLE
feat(file-helper): added an action to convert the encoding

### PR DESCRIPTION
**Pull request by Seb-C**
## What does this PR do?

This adds an action that allows converting the encoding of a file.

I needed that because it is not possible to do that within the `code` action itself, since I could not import any node or browser package from it.

### CODE REVIEW

File: packages/pieces/community/file-helper/src/index.ts [hunk 1]
```diff
@@ -11,6 +12,6 @@ export const filesHelper = createPiece({
   logoUrl: 'https://cdn.activepieces.com/pieces/file-piece.svg',
   categories: [PieceCategory.CORE],
   authors: ['kishanprmr', 'MoShizzle', 'abuaboud'],
```
Comment (AbdulTheActivePiecer) [inline packages/pieces/community/file-helper/src/index.ts]: Add yourself here, so you are shown as a contributor on the piece on our website :D

File: packages/pieces/community/file-helper/src/lib/actions/convert-file.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { Property, createAction } from '@activepieces/pieces-framework';
+import mime from 'mime-types';
+import { encodings } from '../common/encodings';
+
+export const convertFileAction = createAction({
```
Comment (AbdulTheActivePiecer) [inline packages/pieces/community/file-helper/src/lib/actions/convert-file.ts]: Please rename this action to changeFileEncoding

File: packages/pieces/community/file-helper/src/lib/actions/convert-file.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  name: 'convert_file',
+  displayName: 'Convert File',
+  description: 'Changes the encoding of a file',
```
Comment (AbdulTheActivePiecer) [inline packages/pieces/community/file-helper/src/lib/actions/convert-file.ts]: name: 'change_file_encoding',
displayName: 'Change File Encoding'

File: packages/pieces/community/file-helper/src/index.ts (hunk 1 above)
Comment (Seb-C) [reply packages/pieces/community/file-helper/src/index.ts]: Nice, done: https://github.com/activepieces/activepieces/pull/6098/files#diff-cd4c9d71cedac0af6d7e9986944bf179863ada093101a3d5e77ab82f6982919bR14

File: packages/pieces/community/file-helper/src/lib/actions/convert-file.ts (hunk 2 above)
Comment (Seb-C) [reply packages/pieces/community/file-helper/src/lib/actions/convert-file.ts]: Done: https://github.com/activepieces/activepieces/pull/6098/files#diff-4e48b12c0cd432e3012da325a9ef727f529a3665b127c0aaa9a62f28afb02abfR5

File: packages/pieces/community/file-helper/src/lib/actions/convert-file.ts (hunk 3 above)
Comment (Seb-C) [reply packages/pieces/community/file-helper/src/lib/actions/convert-file.ts]: Done: https://github.com/activepieces/activepieces/pull/6098/files#diff-4e48b12c0cd432e3012da325a9ef727f529a3665b127c0aaa9a62f28afb02abfR6-R7

**Pull request merged**